vector/src/main/java/im/vector/app/features/voicebroadcast/usecase/GetOngoingVoiceBroadcastsUseCase.kt

```diff
@@ -31,8 +31,12 @@ class GetOngoingVoiceBroadcastsUseCase @Inject constructor(
 ) {
 
     fun execute(roomId: String): List<VoiceBroadcastEvent> {
-        val session = activeSessionHolder.getSafeActiveSession() ?: return emptyList()
-        val room = session.getRoom(roomId) ?: error("Unknown roomId: $roomId")
+        println("## GetOngoingVoiceBroadcastsUseCase")
```

Comment:
Should we remove this `println`? If we want to keep some logs, we should use `Timber` instead.

Reply:
Good catch... will create a new PR

Reply:
https://github.com/vector-im/element-android/pull/7458